iothub/device/samples/getting started/FileUploadSample/FileUploadSample.csproj

```diff
@@ -27,11 +24,12 @@
     </None>
   </ItemGroup>
 
-  <ItemGroup Condition="'$(Configuration)' == 'Release'">
+  <!--<ItemGroup Condition="'$(Configuration)' == 'Release'">
```

Comment:
Until we're near end of previews or even in GA, we'll need to delay this strategy.